package.json

```diff
@@ -21,6 +20,11 @@
   },
   "dependencies": {
     "@nuxtjs/strapi": "^1.11.0",
-    "vue-tabler-icons": "^2.21.0"
+    "@nuxtjs/i18n": "^8.0.1",
```

Comment:
when i looked at the `@nuxtjs/i18n` docs it seems that it should be a dev dependency. 
![Uploading image.png…]()